### PR TITLE
Add help to modes at Mode tab

### PR DESCRIPTION
**Pull request by HThuren**
This PR add help to Betaflight modes.
When new to Betaflight, you spend many hours to understand the modes, here are some help to reduce that time.
Help text come from betaflight.com where present.
image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]
Nice

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/auxiliary.js
Comment (McGiverGim) [inline src/js/tabs/auxiliary.js:58]: I suggest to use a new variable name and not reuse the `modeName`. And in others places where we "concat" for the messages files, we use the camelize method:
https://github.com/betaflight/betaflight-configurator/blob/8d5533bbd03781f68345532e80ff77ed8910104b/src/js/tabs/osd.js#L2714

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:2290]: I don't know if the `Use to toggle [MODE]` text is necessary. It is just near the name of the mode.  I think only the description is necessary.

**Review by McGiverGim — APPROVED** [review]
Great idea! I didn't know we don't have a description for this.
Only two suggestions, but are not mandatory, only my opinion.

File: src/js/tabs/auxiliary.js
Comment (HThuren) [reply src/js/tabs/auxiliary.js]: Changed

**Pull request merged**
